assets/locales/en/main.json

```diff
@@ -59,9 +59,44 @@
+			"home-button": "Go to the homepage",
+			"sign-in-button": "Sign in again",
+			"reload-button": "Reload the page",
+			"try-again-button": "Try again",
+			"general-help-buttons": "<a target='_blank' href='mailto:hello@streetmix.net'>email</a> or <a target='_blank' href='https://twitter.com/intent/tweet?text=@streetmix'>Twitter</a>. <br /> <button id = 'error-home'onClick = {goHome} > Go to the homepage < /button>"
+		},
+		"page-not-found-title": "Page not found.",
+		"page-not-found-description": "Oh, boy. There is no page with this address!<br />{homeButton}",
+		"street-not-found-title": "Street not found.",
+		"street-not-found-description": "Oh, boy. There is no street with this link!<br />{homeButton}",
+		"street-not-found-but-user-description": "There is no street with this link! But you can look at other streets by {linkToUser(this.props.street)} <br />{homeButton}",
+		"street-deleted-title": "This street has been deleted.",
+		"street-deleted-description": "There is no longer a street with this link, but you can look at other streets by {linkToUser(this.props.street)} <br />{homeButton}",
+		"sign-out-title": "You are now signed out.",
+		"reload-sign-out-title": "You signed out in another window.",
+		"gallery-street-failure-description": "We’re having trouble loading this street.",
+		"reload-sign-out-description-401": "Please reload this page before continuing.<br />(Error RM2.)<br /><button id='error-clear-sign-in-reload' onClick={goReloadClearSignIn}>Reload the page</button>",
+		"reload-sign-in-title": "You signed in in another window.",
+		"reload-sign-in-description": "Please reload this page before continuing.<br />{reloadButton}",
+		"street-deleted-elsewhere-title": "This street has been deleted elsewhere.",
+		"street-deleted-elsewhere-description": "This street has been deleted in another browser.<br />{homeButton}",
+		"twitter-access-denied-title": "You are not signed in.",
+		"twitter-access-denied-description": "You cancelled the Twitter sign in process.<br />{homeButton}",
+		"auth-api-problem-title": "There was a problem with signing you in.",
+		"auth-api-problem-description": "There was a problem with Twitter authentication. Please try again later or let us know via {generalHelpButtons}",
+		"unsupported-browser-title": "Streetmix doesn’t work on your browser.",
+		"unsupported-browser-description": "Sorry about that. You might want to try <a target='_blank' href='http://www.google.com/chrome'>Chrome</a>,&nbsp;<a target='_blank' href='http://www.mozilla.org/firefox'>Firefox</a>,&nbsp;<a target='_blank' href='https://www.microsoft.com/en-us/windows/microsoft-edge'>Microsoft Edge</a>, or Safari.<br /><br />Are you on Internet Explorer?&nbsp;<a target='_blank' href='https://streetmix.readme.io/docs/frequently-asked-questions/#internet-explorer'>Find out more.</a><br /><br />If you think your browser should be supported, please contact us via&nbsp;<a target='_blank' href='mailto:hello@streetmix.net'>email</a>.",
```

Comment:
This is very long and includes a lot of HTML markup, so I think we could make translators' lives easier (and future code maintainer's lives easier) but splitting this description in 3 parts.

1. The "try another browser" sentence.
2. The "Internet Explorer" sentence. This could even be split into 2 more parts, the label and the text link.
3. The "contact us" sentence.

For ease of maintainance, replace links and email (anything inside the `href=` attribute) with a string-replacement placeholder, e.g. `{chromeLink}`.

Reply:
@louh Got it. HTML is already handled by Transifex like this:
![image](https://user-images.githubusercontent.com/12552044/37413685-f821f84a-277d-11e8-9a6d-e56b3bd99436.png)

We could remove it or not. Removing it would create less of those placeholder numbers (the string in screenshot above has a lot of them). How would it affect the code though down the line? Would it help us to keep it in?

Reply:
This really depends. Let's talk about this tomorrow.